modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -34,16 +30,51 @@ Examples:
 }
 ----
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
+
+Using MS2-Example, you can specify multiple examples within a response:
+
+For example, you can have an API specification that contains multiple examples within a response as shown in the following example:
```

Comment:
"For example, ... multiple examples ... as shown in the following example" -> "This example shows three examples for the status code 200:"

Reply:
Changed!